microsoft-sentinel/operations.py

```diff
@@ -132,22 +132,22 @@ def get_all_threat_intelligence_indicators(config, params, connector_info):
 
 
 def get_threat_intelligence_indicator(config, params, connector_info):
-    url = THREAT_INDICATORS_API + "/indicators/{3}?api-version=2022-11-01"
+    url = THREAT_INDICATORS_API + "/indicators/{3}?api-version=2025-09-01"
     endpoint = create_endpoint(config, url, id=params.get('id'))
     response = api_request("GET", endpoint, connector_info, config, params={})
     return response
 
 
 def update_threat_intelligence_indicator(config, params, connector_info):
-    url = THREAT_INDICATORS_API + "/indicators/{3}?api-version=2022-11-01"
+    url = THREAT_INDICATORS_API + "/indicators/{3}?api-version=2025-09-01"
```

Comment:
Please use a constant variable for the API version. This applies to all instances where the 2025-09-01 constant value is used.